I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1729, in script
    $ newDay = dayList.pop(newDayIndex)
  File "game/script.rpy", line 1729, in <module>
    $ newDay = dayList.pop(newDayIndex)
IndexError: pop from empty list

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1729, in script
    $ newDay = dayList.pop(newDayIndex)
  File "C:\Users\Matte\Desktop\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Matte\Desktop\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 1729, in <module>
    $ newDay = dayList.pop(newDayIndex)
  File "C:\Users\Matte\Desktop\renpy-8.1.3-sdk\renpy\revertable.py", line 97, in do_mutation
    return method(self, *args, **kwargs)
IndexError: pop from empty list

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Spoon Theory 1.0
Sat Feb 17 19:30:21 2024
